cloudinit/net/cmdline.py

```diff
@@ -127,6 +127,9 @@ def _klibc_to_config_entry(content, mac_addrs=None):
         else:
             proto = 'static'
 
```

Comment:
Can we instead make this an elif clause so we don't double check a proto that we just setup because it was absent?

```suggestion
elif proto == 'none':
    proto = 'static'
```